cfme/test_requirements.py

```diff
@@ -189,6 +189,12 @@ def test_quota_alert():
     assignee_id='pvala',
 )
 
+ha_proxy = pytest.mark.requirement(
+    "Database HA proxy",
+    description='Databse HA proxy and replication',
+    assignee_id='jhenner',
+)
```

Comment:
@jaryn Should we also remove the `replication` test requirement? Or are there still tests using the `replication` requirement?

Reply:
All tests that are left with replication marker should be mdnadeems. I am sending him a request for renaming/updating the description to say these are centralized-admin tests.

Reply:
@jaryn  Replication FA is owned by Devidas Gaikwad

@dgaikwad , Could you please have a look into it.

Reply:
@jaryn Databse HA proxy and  replication are two different focus area. I think you can remove `replication `string from `ha_proxy `description.

Reply:
> @jaryn Databse HA proxy and replication are two different focus area. I think you can remove `replication `string from `ha_proxy `description.

@dgaikwad For HA you need to have data replicated (https://access.redhat.com/documentation/en-us/red_hat_cloudforms/5.0/html-single/high_availability_guide/index#configuring_primary_db). But by my understanding it is quite different kind of replication than that is done in your focus area which I believe should be named Centralized administration (https://access.redhat.com/documentation/en-us/red_hat_cloudforms/4.7/html-single/deployment_planning_guide/index#central-administration)

Some replication is done in both focus areas.